assets/tests/components/shuttleMapPage.test.tsx

```diff
@@ -71,6 +82,15 @@ describe("Shuttle Map Page", () => {
     expect(result.asFragment()).toMatchSnapshot()
   })
 
+  test("Has the layers control", () => {
+    render(
+      <BrowserRouter>
+        <ShuttleMapPage />
+      </BrowserRouter>
+    )
+    expect(screen.getByRole("button", { name: "Layers" })).toBeInTheDocument()
```

Comment:
suggestion: Since this is replicated across multiple test suites, can you make this into a shared/common selector like the others we have in `assets/tests/testHelpers/selectors`?

Reply:
Great point, updated in https://github.com/mbta/skate/pull/2118/commits/c68c8fc7dd51195bda25c8e5ede6a12f0703add7